packages/composite/dbaas/exoscale/postgres.yml

```
@@ -45,6 +45,7 @@ parameters:
                                   description: MajorVersion is the major version identifier for the
                                     instance.
                                   type: string
+                                  default: 14
```

You probably have to specify it as "14" becuase string :)

```suggestion
                                  default: "14"
```